plugins/inputs/powerdns_recursor/sample.conf

```
@@ -8,3 +8,11 @@
   # socket_dir = "/var/run/"
   ## Socket permissions for the receive socket.
   # socket_mode = "0666"
+
+  ## The version of the powerdns control protocol to use. You will have to
+  ## change this based on your PowerDNS Recursor version, see below:
+  ## PowerDNS <4.5.0 => Protocol Version 1
+  ## PowerDNS 4.5.0 - 4.5.9 => Protocol Version 2
+  ## PowerDNS >=4.6.0 => Protocol Version 3
```

```suggestion
  ## less than version 4.5.0 => Protocol Version 1
  ## 4.5.0 through 4.5.9 => Protocol Version 2
  ## greater than or equal to version 4.6.0 => Protocol Version 3
```

I was thinking maybe this would be clearer? If do agree this is a good change, don't forget to run `make docs`

---

IMHO the well-known symbols `<>=` are clearer and easier to read, especially for the group of people using telegraf.
Writing it all out makes it more convoluted and harder to navigate over with your eyes.